custom_components/ha-glocaltokens/utils.py

```diff
@@ -0,0 +1,5 @@
+"""utils.py for google home integration"""
+
+
+def convert_from_ms_to_s(timestamp):
+    return timestamp / 1000
```

Comment:
That's great! Having utils file is convenient. I think we should round this number to the closest integer or if it's float then to something with a limited about of digits after the comma. 
What I am afraid of ending up with `convert_from_ms_to_s(...)` return something like `3.781637163` seconds. We can either round it up to the closest integer like `4` or just leave two decimals after the comma and return `3.78`. What do you think?

Reply:
because it's unix timstamp i think we should avoid having decimals